tests/ops/functions/test_is_commuting.py

```diff
@@ -170,28 +170,11 @@ def test_cnot_cz(self, wires, res):
     def test_cz_mcz(self, wires, res):
         """Commutation between CZ and MCZ."""
 
-        def z():
-            qml.PauliZ(wires=wires[1][2])
+        op1 = qml.ctrl(qml.PauliZ(wires=wires[1][2]), control=wires[1][:-1])
```

Comment:
Maybe use `wires[1][-1]` instead of `wires[1][2]` to make sure that `PauliZ` and `control` don't have overlapping wires.

Reply:
done.